src/main/java/ddingdong/ddingdongBE/domain/formapplication/entity/FormApplication.java

```diff
@@ -41,4 +41,8 @@ private FormApplication(String name, String studentNumber, String department, Fo
         this.status = status;
         this.form = form;
     }
+
+    public void update(FormApplicationStatus status) {
+        this.status = status;
+    }
```

Comment:
p3)
해당 메서드가 어떤 FormApplication의 무엇을 업데이트하는 메서드인지 명확하게 표현하는것이 좋을거 같습니다!
```suggestion
    public void updateFormApplicationStatus (FormApplicationStatus status) {
        this.status = status;
    }
```

Reply:
updateStatus()로 변경했습니다! 감사합니다!